setup.cfg

```diff
@@ -1,3 +1,64 @@
+[metadata]
+name = awkward
+url = https://github.com/scikit-hep/awkward-1.0
+project_urls =
+  Documentation = https://awkward-array.org
+  Source Code = https://github.com/scikit-hep/awkward-1.0
+  Bug Tracker = https://github.com/scikit-hep/awkward-1.0/issues
+  Discussions = https://github.com/scikit-hep/awkward-1.0/discussions
+  Chat = https://gitter.im/Scikit-HEP/awkward-array
+  Releases = https://github.com/scikit-hep/awkward-1.0/releases
+author = Jim Pivarski
+author_email = pivarski@princeton.edu
+maintainer = Scikit-HEP
+maintainer_email = scikit-hep-admins@googlegroups.com
```

Comment:
@jpivarski Do you want to be listed as the maintainer, or would you like to put the standard Scikit-HEP snippet here? This is what the Scikit-HEP one looks like, can do either one.

Reply:
References to my name as email and any copyright messages should transition toward Scikit-HEP. Since there's a standard email, let's go with it. Thanks!